Files/Extensions/DateTimeExtensions.cs

```diff
@@ -66,15 +66,16 @@ public static (string text, string range, string glyph, int index) GetFriendlyTi
             Windows.Globalization.Calendar cal = new Windows.Globalization.Calendar();
             var t = DateTimeOffset.Now;
             var t2 = dt.ToLocalTime();
+            var y = t.AddDays(-1);
             var today = DateTime.Today;
 
             var diff = t - dt;
-            if (t.Month == t2.Month && t.Day == t2.Day)
+            if (t.Month == t2.Month && t.Day == t2.Day && t.Year == t2.Year)
             {
                 return ("ItemTimeText_Today".GetLocalized(), today.ToUserDateString(), "\ue184", 0);
             }
 
-            if (t.Month == t2.Month && t.Day - t2.Day < 2)
+            if (y.Month == t2.Month && y.Day == t2.Day && y.Year == t2.Year)
```

Comment:
also what's the reason for replacing the `t.Day - t2.Day < 2` with `y.Day == t2.Day`? This section is meant to show items from the previous day, hence why it checks that it's not more then two days apart.

Reply:
it doesn't go well when month changes, for that I created "y" to be "Now-1 day" and compare that instead
`
var t  = new DateTime(2021,12,01,01,02,03);
var t2 = new DateTime(2021,11,29,23,02,03);
Console.WriteLine(t.Day - t2.Day);
`
(output is -28)